sys/include/net/skald.h

```diff
@@ -55,17 +55,56 @@ extern "C" {
 #endif
 
 /**
- * @brief   Static advertising interval
+ * @defgroup net_skald_conf Skald compile configurations
+ * @ingroup config
+ * @{
+ */
+/**
+ * @brief   Advertising interval in microseconds
+ */
+#ifndef CONFIG_SKALD_INTERVAL
+#define CONFIG_SKALD_INTERVAL          (1 * US_PER_SEC)
+#endif
+
+/**
+ * @brief   Configure advertising channel 37
+ *
+ * Set CONFIG_ADV_CH_37_DISABLE to disable channel 37
  */
-#ifndef SKALD_INTERVAL
-#define SKALD_INTERVAL          (1 * US_PER_SEC)
+#ifndef CONFIG_ADV_CH_37_DISABLE
+#define ADV_CH_37 37,
```

Comment:
I don't think the `ADV_CH_n` macros should be part of the compile configuration group, as they are not to be changed by the user directly, right?

On the other hand, `CONFIG_ADV_CH_n_DISABLE` macros _should_ appear on the documentation page.

What do you think?

Reply:
Yes. That should work. I shall also update the documentation of SKALD_ADV_CHAN just in case.

Reply:
But wait. I am not defining CONFIG_ADV_CH_n_DISABLE macro in the code but via Kconfig. Should I skip the macros altogether?

Reply:
You can use `#ifdef DOXYGEN` to add the macro definitions and documentation only when building the docs

Reply:
Although `ADC_CH_n` macros are not part of the group they need documentation, otherwise `make doc` complains...

Reply:
Ah yes. The webpage was rendered correctly though. 
Updated